deploy/cloud/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Local .terraform directories
```

Comment:
you can add this to the original gitignore

Reply:
I think it's nicer to have multiple separate `.gitignore` files that are more specific